internal/api/v1/user/configs/configs.go

```diff
@@ -0,0 +1,43 @@
+package configs
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/bitcoin-sv/spv-wallet/models"
+	"github.com/bitcoin-sv/spv-wallet/models/response"
+	"github.com/go-resty/resty/v2"
+)
+
+const route = "api/v1/configs"
+
+type API struct {
+	addr string
+	cli  *resty.Client
+}
+
+func (a *API) SharedConfig(ctx context.Context) (*response.SharedConfig, error) {
+	var spvErr models.SPVError
+	var result response.SharedConfig
+
+	resp, err := a.cli.
+		R().
+		SetContext(ctx).
+		SetResult(&result).
+		SetError(&spvErr).
+		Get(a.addr + "/shared")
+	if err != nil {
+		return nil, fmt.Errorf("HTTP error: %w", err)
+	}
+	if resp.IsError() {
+		return nil, spvErr
+	}
```

Comment:
If error returned by `spv-wallet` was not "standard SPVError" (json) but rather a raw "string" (sometimes it should happen) it would return "empty SPVerror". I mean struct with zeroed data.

Maybe sth like that would solve this problem.
```golang
if resp.IsError() {
        if spvErr.Code == "" {
            return nil, fmt.Errorf("Non-standard error returned by spv-wallet") //text msg should be adjusted :D 
        }
		return nil, spvErr
}
```